ftva_lab_data/views_utils.py

```diff
@@ -20,6 +20,9 @@ def get_field_value(obj: models.Model, field: str) -> Any:
     Returns the value of the field, or an empty string if the field does not exist
     or is None.
     """
+    # Special case for status field, which is a ManyToMany field
+    if field == "status":
+        return [str(status) for status in obj.status.all()]
```

Comment:
Unless Thelma has said these should be in a specific (non-alphabetical) order, I think these should be alphabetized.  Otherwise, these seem to be ordered by `ItemStatus.id`.